source/features/table-input.css

```
@@ -0,0 +1,27 @@
+.rgh-add-table-popup {
+	width: 108px;
+	height: 108px;
+	padding: 4px;
+	z-index: 99;
+	display: grid;
+	grid-template: repeat(5, 1fr) / repeat(5, 1fr);
+	place-content: center;
+}
+
+.rgh-add-table-popup__square {
+	appearance: none;
+	background: none;
+	border: none;
+	padding: 2px;
+}
+
+.rgh-add-table-popup__square > div {
```

There's no need for a `div`
```suggestion
	margin: 2px;
```

---

If you set a margin directly on the buttons, the user might click between the grid cells and nothing will happen, which is annoying. Also the table size feedback will feel slightly choppy.

By having buttons stuck together with no gaps + decorative divs, we can get a smooth UI while keeping the visual clarity and aesthetic of separate cells. Sure it's a detail but I think it matters :smile:

---

Good point, it does matter!